docs/ProcessingPipeline.md

```diff
@@ -0,0 +1,62 @@
+# Processing Pipeline
+
+Some applications might require that traces be altered or filtered out before they are sent upstream. The processing pipeline allows users to create *processors* to define such behavior.
```

Comment:
Minor:

1. `upstream` => to Datadog?
2. "allows users" seems to be written from a library developer POV, I suggest perhaps "allows you" or "allows those applications"?